sympy/integrals/transforms.py

```diff
@@ -1939,6 +1939,46 @@ def laplace_transform(f, t, s, legacy_matrix=True, **hints):
 
 
 @_noconds_(True)
+
+def _inverse_laplace_build_rules(s, t):
```

Comment:
Why is the `_noconds_` decorator being used here?

Reply:
Because I inserted my code in the wrong place.  Thanks!  This solves my problem.